README.md

```diff
@@ -19,9 +19,39 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
 
+[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
 ### How to Include on Your Platform
 
-TODO
+
+**Javascript**
```

Comment:
Looks pretty good but can you clarify more regarding cloning the project first then where to find the js output folder ?

Reply:
@mostafashaaban5 thanks. I have added the Information to the js specific project

Should I replicate that here?

Reply:
@mostafashaaban5 I have added the Information to the js specific project.

Should I replicate that here?

Reply:
I think would be better

Reply:
Alright